build.gradle

```diff
@@ -372,9 +385,38 @@ task createApiDocsRedirect {
   }
 }
 
+task publishDocs {
+  dependsOn 'closeAndReleaseRepository'
+  doLast {
```

Comment:
This is the release process for gax so we don't need to worry about the batch generation :)
For other packages such as gRPC which has the batch use-case, I believe we would have something setup in the root project of api-client-staging and iterate each sub-project's release task. Confirm with @michaelbausor